lib/salus/report.rb

```diff
@@ -40,6 +42,17 @@ def initialize(report_uris: [], builds: {}, project_name: nil, custom_info: nil,
       @ignore_config_id = ignore_config_id # ignore id in salus config
     end
 
+    def apply_report_hash_filters(report_hash)
+      @@filters.each do |filter|
+        report_hash = filter.filter_config(report_hash) if filter.respond_to?(:filter_config)
```

Comment:
We will want to avoiding overriding filter_config (that hook is for filtering the config).  We should use filter_report_hash here:

```
def apply_report_hash_filters(report_hash)
  @@filters.each do |filter|
    report_hash = filter.filter_report_hash(report_hash) if filter.respond_to?(:filter_report_hash)
  end
  report_hash
end
```
That will ensure that we have different hooks for overriding configs ( filter_config ) and overriding reports hashes ( filter_report_hash )

Reply:
Thanks, updated accordingly.